### PR TITLE
merge whitelisted, whitelist-add, and whitelist-remove

### PR DESCRIPTION
**Pull request by Susideur**

### CODE REVIEW

**Review by Quezler — CHANGES_REQUESTED** [review]
the coding style is wrong, too much indentation and spaces around ifs among other things 👀

**Comment by genNAowl** [comment]
Check https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md I guess?

**Comment by Susideur** [comment]
chit nadly indentations 
I correct her

**Comment by Anuken** [comment]
The indentation and formatting is still wildly incorrect, look at the surrounding code and the contribution guide linked above.

File: server/src/mindustry/server/ServerControl.java [hunk 1]
```diff
@@ -600,36 +600,42 @@ protected void registerCommands(){
...
-            netServer.admins.whitelist(arg[0]);
-            info("Player '@' has been whitelisted.", info.lastName);
-        });
-
-        handler.register("whitelist-remove", "<ID>", "Remove a player to the whitelist by ID.", arg -> {
-            PlayerInfo info = netServer.admins.getInfoOptional(arg[0]);
-            if(info == null){
-                err("Player ID not found. You must use the ID displayed when a player joins a server.");
-                return;
+        handler.register("whitelist", "[add/remove] [ID]", "Only members of the whitelist can connect to the server.", arg -> {
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java:603]: The description doesn't explain what the command does.

File: server/src/mindustry/server/ServerControl.java [hunk 2]
```diff
@@ -600,36 +600,42 @@ protected void registerCommands(){
...
+            if (arg.length == 0) {
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java:604]: formatting still incorrect

File: server/src/mindustry/server/ServerControl.java [hunk 3]
```diff
@@ -600,36 +600,42 @@ protected void registerCommands(){
...
+                    whitelist.each(p -> Log.info("- Name: @ / UUID: @", p.lastName, p.id));
+                }
+            }else{
+                if(arg.length == 2){
+                    PlayerInfo info = netServer.admins.getInfoOptional(arg[1]);
+        		    
+                    if(info == null){
+                        Log.err("Player ID not found. You must use the ID displayed when a player joins a server.");
+                    }else{
+                        switch(arg[0]){
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java:620]: Un-enhanced switch statements with only two cases are too verbose, use ifs instead.

File: server/src/mindustry/server/ServerControl.java [hunk 4]
```diff
@@ -600,36 +600,42 @@ protected void registerCommands(){
...
+                                netServer.admins.unwhitelist(arg[1]);
+                                Log.info("Player '@' has been un-whitelisted.", info.lastName);
+                                break;
+        				
+                            default:
+                                Log.err("Invalid arguments");
+                        }
+                    }
+                }else{
+                    Log.err("Invalid arguments");
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java:636]: Explain the actual issue, e.g. "Missing ID argument". Use a period.

**Review by Anuken — CHANGES_REQUESTED** [review]

**Comment by Susideur** [comment]
it's good ?

**Comment by WilloIzCitron** [comment]
hello, this pull is stale?

**Comment by Anuken** [comment]
There are still several issues with this PR, so I decided it was faster to finish up the changes myself. See linked commit.

**Pull request closed without merging**